### ISSUE TITLE
Questions Regarding Training Video Generation Model

### ISSUE THREAD

**Issue by WangYixuan12** [ShuangLI59/unified_video_action#2]
Hi @ShuangLI59, congrats on releasing this amazing work! I quickly try out training video generation model on PushT task using Cheng's PushT dataset. I run the following command on two Nvidia L40 GPUs for more than 12 hours.
```
CUDA_VISIBLE_DEVICES=2,3 accelerate launch --num_processes=2 train.py \
    --config-dir=. \
    --config-name=uva_pusht.yaml \
    model.policy.action_model_params.predict_action=False \
    model.policy.selected_training_mode=video_model \q
    model.policy.optimizer.learning_rate=1e-4 \
    logging.project=uva \
    hydra.run.dir="checkpoints/uva_pusht_video_model"
```
However, I obtained the following video generation results, which is a bit weird.

https://github.com/user-attachments/assets/6cbf8424-2122-41b4-b8a9-e19b41fe7fb1

Additionally, the training loss seems to plateau as well.

![Image](https://github.com/user-attachments/assets/ce4f786c-290a-4d3a-910a-ca5b3e880f80)

I am wondering whether this is expected. Is my video generation model training also correct?

**Comment by Cytuser** [comment]
> Thank you for prompt response! I just realized that I forgot to download pretrained models, which might result in training from scratch. After downloading pretrained models, now it looks great! Great works!

Glad to hear it's working now! Could you tell me which pretrained model you used?

**Comment by WangYixuan12** [comment]
https://drive.google.com/file/d/1OduHcxfc2hqUYSccMQNf9g-vAt-q2UhF/view?usp=sharing
